src/features/presence/utils/presenceUtils.js

```diff
@@ -60,4 +64,24 @@ function determineInteractionMessage(interactionType, interactedAt) {
     return message;
 }
 
-export { sortByActivity, getLastActionTimeMS, determineInteractionMessage };
+function convertMillisecondsToUnitAndValue(ms) {
+    const seconds = ms / 1000;
```

Comment:
could we keep it in milliseconds instead of converting to seconds? I think for consistency since the other functions use milliseconds

as well as the constants can stay in the same unit:

```
const MS_IN_A_SECOND = 1000;
const MS_IN_A_MINUTE = MS_IN_A_SECOND * 60;
const MS_IN_A_HOUR = MS_IN_A_MINUTE * 60;
const MS_IN_A_DAY = MS_IN_A_HOUR * 24;
const MS_IN_A_MONTH = MS_IN_A_DAY * 30;
```